packages/manager/apps/public-cloud/src/assets/theme/dark/index.less

```diff
@@ -1000,5 +1000,26 @@
   manager-error-page .ovh-manager-error-page {
     background: #181818;
   }
+
+  /* hub navbar */
+  .oui-navbar-menu-hub {
+    @notification-bcg-color: #def8ff;
+
+    background-color: @notification-bcg-color;
+    border: 1px solid @notification-bcg-color;
+
+    .oui-navbar-notification {
+      .oui-status_error {
+        color: #c11b1b;
+      }
+      .oui-status_info {
+        color: #4d6f92;
+      }
+      a.oui-navbar-notification__content,
+      a.oui-navbar-notification__content a {
```

Comment:
Nested `<a>` selector? 🤔 

Is there an issue with the component template?

Reply:
yes, HTML has nested <a> tag

Reply:
@antleblanc there is no issue with "a.oui-navbar-notification__content a". It required because HTML has nested anchor tag. I verified and it is working properly.

Reply:
@radireddy Having nested anchor tags is an issue. This is not valid.
We have to fix the navbar notification template first.
Could you please have a look? Thanks!

@marie-j Is this template has recently been modified to remove this nested anchor tags issue?

Reply:
@antleblanc The issue is the component is an anchor tag but we sometimes have anchor tags in the content returned by 2API notification (not always but for one or two)